backend/tests/services/knowledge/structured/test_sql_validator.py

```
@@ -0,0 +1,178 @@
+
+        assert result["is_valid"] is True
+
+    def test_cte_query(self):
+        """Test Common Table Expression (CTE) queries."""
+        sql = """
+        WITH top_items AS (
+            SELECT * FROM kb_1_doc_1 WHERE amount > 100
+        )
+        SELECT * FROM top_items ORDER BY amount DESC LIMIT 10
+        """
+        result = self.validator.validate(sql, self.allowed_tables)
```

Ruff (0.15.0)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)